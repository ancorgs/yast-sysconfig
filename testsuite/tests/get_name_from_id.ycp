// Sysconfig module - testsuite
//
// get_name_from_id tests
//
// $Id$
//

{

include "testsuite.ycp";

import "Sysconfig";

TEST( ``(Sysconfig::get_name_from_id(nil) ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("var") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("var$file") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("var$") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("$file") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("var$x$file") ), [ $[], $[], $[] ], nil );
TEST( ``(Sysconfig::get_name_from_id("$$varx$file") ), [ $[], $[], $[] ], nil );

}

