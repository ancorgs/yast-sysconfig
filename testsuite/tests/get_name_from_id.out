Return	
Return	
Return	
Return	var
Return	var
Return	
Return	var
Return	
